src/pages/homepage/Homepage.js

```diff
@@ -0,0 +1,18 @@
+import { QuizCard } from 'component/card/QuizCard'
+import React from 'react'
+import '../../component/card/quizcard.css'
```

Comment:
You can implement barrel imports and exports for better readability https://dev.to/justmyrealname/organize-react-components-better-with-barrel-exports-4fno